client/src/pages/SignIn.js

```diff
@@ -0,0 +1,158 @@
+import React, { useState } from "react";
+import {
+  Button,
+  Typography,
+  TextField,
+  Card,
+  CardContent,
+  Grid,
+} from "@material-ui/core";
+import { Link } from "react-router-dom";
+import { withStyles } from "@material-ui/core/styles";
+import BackgroundImg from "../Assets/images/signUpBkg.png";
+import ChefsMenuLogo from "../Assets/images/Logo.png";
+
+const signInPageStyle = (theme) => ({
+  signInContainer: {
+    fontFamily: '"Montserrat"',
+    margin: theme.spacing(2),
```

Comment:
If you move this line from this class to the titleLogo class, it will fix the weird border on the sides. Same on the SignUp page